packages/dds/task-manager/src/interfaces.ts

```diff
@@ -5,19 +5,126 @@
 
 import { ISharedObject, ISharedObjectEvents } from "@fluidframework/shared-object-base";
 
+/**
+ * Describes the event listener format for {@link ITaskManagerEvents} events.
+ *
+ * @param taskId - The unique identifier of the related task.
+ */
+export type TaskEventListener = (taskId: string) => void;
+
 /**
  * Events emitted by {@link TaskManager}.
  */
 export interface ITaskManagerEvents extends ISharedObjectEvents {
 	/**
-	 * Notifies when the local client has reached the front of the queue, left the queue, or a task was completed.
-	 * Does not account for known pending ops, but instead only reflects the current state.
+	 * Notifies when the specified task has been exclusively assigned to the client.
```

Comment:
```suggestion
	 * Fires when a task has been exclusively assigned to the client.
```
Something to note - we don't setup the listener for each individual task. We listen to the even on TaskManager and it will fire for each and every task that is assigned to the client (and will specify the task with `taskId`)

Reply:
My phrasing wasn't great - I meant to indicate that the event fired _with_ the task ID of the relevant task. But I like your wording, and the type alias for the listener already includes notes about the task ID.